src/iterator.rs

```diff
@@ -240,7 +256,7 @@ where
                     }) {
                         // TODO(nnmm): This crashes on error. We should bubble up an error.
```

Comment:
while reformatting, should we get rid also of this todo as well?

Reply:
It's already gone, this is solved by that change :)